pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -24,12 +24,20 @@ stdenv.mkDerivation rec {
   '';
 
   # Fix lib extension so that fixDarwinDylibNames detects it
-  postInstall = lib.optionalString stdenv.isDarwin ''
+  postInstall = lib.optionalString (stdenv.isDarwin && !stdenv.isAarch64) ''
     mv $lib/lib/liblowdown.{so,dylib}
   '';
 
   patches = lib.optional (!stdenv.hostPlatform.isStatic) ./shared.patch;
 
+  patchPhase = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) ''
+    substituteInPlace main.c \
```

Comment:
You should be able to set `HAVE_SANDBOX_INIT=0` in `configure.local` instead which I'd prefer as it is the intended way setting this (and doesn't require us to touch the real source code).

Reply:
I tried that, but thinking back, I probably didn't set this env-var right. Let me try if I get better luck.

Reply:
configure.local did the trick, I'm amazed 👍